src/layouts/index.html

```diff
@@ -15,6 +15,33 @@ <h2>Learn about Giant Swarm</h2>
         </li>
       </ul>
     </div>
+    <div class="col-sm-4">
+      <ul class="linklist">
+        <li>
+          <a href="/basics/aws-architecture/">AWS Architecture</a>
+          <div class="meta">How Giant Swarm is running Kubernetes clusters based on AWS resources</div>
```

Comment:
"based on AWS resources" reads a bit strangely to me. What about "running K8s clusters on AWS infrastructure"?